source/glossary-terminology.rst

```diff
@@ -1,353 +1,55 @@
-
-      A series of data which are visualized in |pmm|.
-
-.. _Metrics-Monitor:
-
-`Metrics Monitor (MM) <glossary-terminology.html#metrics-monitor>`_
---------------------------------------------------------------------------------   
-
-      Component of :ref:`PMM-Server` that provides a historical view of
-      :ref:`metrics <Metrics>` critical to a |mysql| server instance.
-
-.. _Monitoring-service:
-
-`Monitoring service <glossary-terminology.html#monitoring-service>`_
---------------------------------------------------------------------------------
-
-      A special service which collects information from the database instance
-      where :ref:`PMM-Client` is installed.
-
-      To add a monitoring service, use the :program:`pmm-admin add` command.
-
-      .. seealso::
-
-	 Passing parameters to a monitoring service
-	    :ref:`pmm.pmm-admin.monitoring-service.pass-parameter`
-
-.. _PMM:
-
-`PMM <glossary-terminology.html#pmm>`_
---------------------------------------------------------------------------------
-
-      Percona Monitoring and Management
-
-.. _pmm-admin:
-
-`pmm-admin <glossary-terminology.html#pmm-admin>`_
---------------------------------------------------------------------------------
-
-      A program which changes the configuration of the :ref:`PMM-Client`. See
-      detailed documentation in the :ref:`pmm-admin` section.
-
-.. only:: showhidden
-
-
-	 .. _PMM-annotation:
-
-	 `PMM annotation <glossary-terminology.html#pmm-annotation>`_
-	 --------------------------------------------------------------------------------
-
-	   A feature of |pmm-server| which adds a special mark to all
-	   dashboards and signifies an important event in your
-	   application. Annotations are added on the |pmm-client| by using
-	   the |pmm-admin.annotate| command.
-
-	   .. seealso::
-
-		 |grafana| Documentation: Annotations
-
-		    http://docs.grafana.org/reference/annotations/
-
-.. _PMM-Client:
-
-`PMM Client <glossary-terminology.html#pmm-client>`_
---------------------------------------------------------------------------------   
-
-      Collects |mysql| server metrics, general system metrics,
-      and query analytics data for a complete performance overview.
-
-      The collected data is sent to :ref:`PMM-Server`.
-
-      For more information, see :ref:`pmm.architecture`.
-
-.. _PMM-Docker-Image:
-
-`PMM Docker Image <glossary-terminology.html#pmm-docker-image>`_
---------------------------------------------------------------------------------
-
-      A docker image which enables installing the |pmm-server| by
-      using :program:`docker`.
-
-      .. seealso::
-
-	 Installing |pmm-server| using |docker|
-	    :ref:`run-server-docker`
+.. CREATED BY make_glossary.pl - DO NOT EDIT!
 
-.. _PMM-Home-Page:
+.. raw:: html
 
-`PMM Home Page <glossary-terminology.html#pmm-home-page>`_
---------------------------------------------------------------------------------
+    <!-- Local style adjustments -->
```

Comment:
This overrides the Percona style for <dl> and <dd> elements to make glossary items easier to read.